### PR TITLE
Particle effect component doc

### PR DESCRIPTION
**Pull request by r41k0u**

### CODE REVIEW

File: docs/guides/effects_using_ParticleEffectComponent.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+You can follow the `documentation <https://effekseer.github.io/en/documentation.html>`_ and `tutorial <https://effekseer.github.io/en/documentation.html#tutorial_sec>`_ of Effekseer to know how to create effects using it.
+
+Using the Effekseer exports (Demo)
+==================================
+
+Let's start by creating an empty scene and adding the TransformComponent and ParticleEffectComponent to it.
+
+I have added a demo effect which we get bundled with Effekseer 1.62b (Laser01.efkefc)
+Add the effect by going to your scene's ParticleEffectComponent in the Inspector aand clicking on the folder icon.
+I have tweaked the `EditorCamera <https://rootex.readthedocs.io/en/latest/guides/editor_layout.html#editorcamera>`_ to get a better view of the scene.
```
Comment (Ishan-002) [inline docs/guides/effects_using_ParticleEffectComponent.rst]: Change ```I``` to ```We```

File: docs/guides/effects_using_ParticleEffectComponent.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Explanation of options:
+* Play : Plays the loaded effect
+* Stop : Stops the playing effect
+* Play On Start : Sets whether to play the effect at editor and game start or not.
+* Start Frame : Sets the frame from which effect starts.
+* Moving : Sets if the effect moves with the transform component once playing.
+* Use Speed : Running the effect at a user defined spped. Default speed is 1.
+
+While Play, Stop and Play On Start are pretty self-explanatory, The follwoing explains the other options.
```
Comment (Ishan-002) [inline docs/guides/effects_using_ParticleEffectComponent.rst]: follwoing -> following

**Review by Ishan-002 — CHANGES_REQUESTED** [review]

**Review by Ishan-002 — COMMENTED** [review]
None of the images/gifs are getting rendered. 
image
Try to build the tutorial locally and then check it once.

File: docs/guides/getting_started.rst [hunk 3]
```diff
@@ -15,4 +15,7 @@ Any user coming from ECS based game engines should be able to pick up the interf
 
 	running_the_editor
 	editor_layout
+	graphics_tutorial
+	making_HUD_using_ui-component
```
Comment (Ishan-002) [inline docs/guides/getting_started.rst]: remove these

File: docs/guides/effects_using_ParticleEffectComponent.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
+============================================
+Making effects using ParticleEffectComponent
+============================================
+
+ParticleEffectComponent in Rootex can be used to make awesome particle effects!
+
+Effekseer
+=========
```
Comment (angad-k) [inline docs/guides/effects_using_ParticleEffectComponent.rst]: Use `---` to denote sections and `~~~` to denote subsections.

File: docs/guides/effects_using_ParticleEffectComponent.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+This ``*.efkefc`` file is used by the ParticleEffectComponent in Rootex to import the effects.
+
+You can follow the `documentation <https://effekseer.github.io/en/documentation.html>`_ and `tutorial <https://effekseer.github.io/en/documentation.html#tutorial_sec>`_ of Effekseer to know how to create effects using it.
+
+Using the Effekseer exports (Demo)
+==================================
+
+Let's start by creating an empty scene and adding the TransformComponent and ParticleEffectComponent to it.
+
+We have added a demo effect which we get bundled with Effekseer 1.62b (Laser01.efkefc)
```
Comment (Ishan-002) [inline docs/guides/effects_using_ParticleEffectComponent.rst]: Add the link for Laser01.efkefc. I myself couldn't find it.

File: docs/guides/effects_using_ParticleEffectComponent.rst [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+Add the effect by going to your scene's ParticleEffectComponent in the Inspector and clicking on the folder icon.
```
Comment (Ishan-002) [inline docs/guides/effects_using_ParticleEffectComponent.rst]: Maybe change ```folder icon``` to ```folder icon to select the file```

**Review by Ishan-002 — CHANGES_REQUESTED** [review]
There are some spacing problems in headings and after some images

**Review by Ishan-002 — APPROVED** [review]

**Pull request merged**